### DISCUSSION TITLE
How to filter cal::local_date in respect to current date?

### DISCUSSION THREAD

**Discussion by divan** [geldata/gel#2705] [Q&A] [answered]
So let's say we have table `Events` with a property `date -> cal::local_date`.
I need to select future and past events, i.e. events where date is less or more than current date.

I see in [docs](https://www.edgedb.com/docs/edgeql/funcops/datetime#function::std::datetime_current) the function `datetime_current()`, but it seems there is no convenient method to cast/convert it to local_date.

The question is – how to filter local_date field using datetime_current? I don't care at this point about timezone of the date, so it's ok to just convert with the current server timezone into date.

**Comment by divan** [comment, marked as answer]
Okay, figured it out myself:

```FILTER .date < cal::to_local_date(datetime_current(), "UTC")```

**Reply by 1st1** [reply]
Yep! You can also do the same trick in reverse with `to_datetime()`.

How do you find working with EdgeDB? Do you like the documentation?

**Reply by divan** [reply]
Totally! It's beautiful.
